fix docs: add guide for update docs (#1359)
oam-dev/kubevela: CONTRIBUTING.md

make e2e-test
```

> ### Docs

Reviewer: ```suggestion
### Contribute Docs
```

> Please read [the documentation](https://github.com/oam-dev/kubevela/tree/master/docs/README.md) before modifying the docs.

Reviewer: ```suggestion
Please read [the documentation](https://github.com/oam-dev/kubevela/tree/master/docs/README.md) before contributing to the docs.
```

- Build docs

```shell script
make docs-build
```

> - Local development

Reviewer: ```suggestion
- Local development and preview
```
